### ISSUE TITLE
Need directions

### ISSUE THREAD

**Issue by repcosrvces** [rlauzon54/Kaypro-4-83#1]
I have a kaypro IV i want to add gotek to. So glad to find your work. I can't find instructions for making image files of disks. Hiw did you accomplish it?

Well  i goofed, because it wasn't very clear. Mud may be clearer. But just formatting the thumb drive and loading your hfe files then adding an OLED and the gotek says good stuff to me. It worked!  I still need to test it, and find a way to make my own images, as i prefer newword to wordstar. (doesn't matter actually)

**Comment by rlauzon54** [comment]
I never created a disk image from scratch.  I used the existing disk images that can be found on the Internet for the Kaypro.

Then I just used the Kaypro programs (like `COPY`) to copy the diskettes I wanted, depending on what I wanted to do.,

**Comment by repcosrvces** [comment]
ok, I did find 2 sites after leaving you that note. But they were not as clear in their labels as you.  I'm not sure that the process is very clear. But, it seems that these images are supposed to be an "image" of a disk having format, sectors, files, etc. Is that correct? The intent , I gather, is to just copy the image into one of the gotek image files created by formatting the thumb drive. However, my gotek gives me an error even when the image files are blank. E34  not able to navigate.  It isn't clear what to do about it.

**Comment by rlauzon54** [comment]
I think I see your problem.  A Gotek doesn't work "out of the box".  You need to re-flash it with the FlashFloppy firmware first.

There's something called OpenFlops, which is basically an open source Gotek but already has FlashFloppy loaded.  That's what I usually use - just because I don't have to bother to flash the firmware on a stock Gotek.

But once FlashFloppy is loaded, you just need to set the FF.CFG file:
interface=shugart
host=kaypro
nav-mode=native

and drop the disk images on it.

**Comment by repcosrvces** [comment]
yes that did work, I had already used flash floppy and they kind of leave off there. "load images..." but how to make images???? So glad you had good ones, and only thing i added was a kaypro.cfg ... if it was needed or not I don't know.  but it worked !

**Comment by repcosrvces** [comment]
have you tried the unbranded gotek clones?

**Comment by rlauzon54** [comment]
The config file needs to be "ff.cfg" for the FlashFloppy firmware to see it.
I mostly use the disk images from this site: http://www.retroarchive.org/maslin/disks/kaypro/
At worst, you can reformat it in the Kaypro for something else.

I did try unbranded Goteks.  They work fine.  They vary only in what headers they have pre-soldered.  But they all work.
